plugin/pkg/scheduler/algorithm/well_known_labels.go

```diff
@@ -28,6 +28,26 @@ const (
 	// and removed when node becomes reachable (NodeReady status ConditionTrue).
 	TaintNodeUnreachable = "node.alpha.kubernetes.io/unreachable"
 
+	// When feature-gate for TaintBasedEvictions=true flag is enabled,
+	// TaintNodeOutOfDisk would be automatically added by node controller
+	// when node becomes out of disk, and removed when node has enough disk.
+	TaintNodeOutOfDisk = "node.kubernetes.io/outOfDisk"
```

Comment:
I'd like to update other taint when graduating from alpha to beta in separate PR.